sklearn/datasets/_base.py

```diff
@@ -1096,6 +1096,23 @@ def load_diabetes(*, return_X_y=False, as_frame=False, scaled=True):
         representing the features and/or target of a given sample.
 
         .. versionadded:: 0.18
+
+    Examples
+    --------
+    To load the data and extract the information about the samples, features names
+    and the shape of the traget variable::
+
+        >>> from sklearn.datasets import load_diabetes
+        >>> diabetes = load_diabetes()
+        >>> data, target = diabetes.data, diabetes.target
+        >>> print(f"Number of samples: {data.shape[0]}")
+        Number of samples: 442
+        >>> print(f"Number of features: {data.shape[1]}")
+        Number of features: 10
+        >>> print(f"Feature names: {diabetes.feature_names}")
+        Feature names: ['age', 'sex', 'bmi', 'bp', 's1', 's2', 's3', 's4', 's5', 's6']
+        >>> print(f"Target shape: {target.shape}")
+        Target shape: (442,)
```

Comment:
Let's simplify as much as possible. We don't need any explanation here.

```suggestion
    >>> from sklearn.datasets import load_diabetes
    >>> diabetes = load_diabetes()
    >>> diabetes.target[:3]
    array([151.,  75., 141.])
    >>> diabetes.data.shape
    (442, 10)
```

Reply:
Okay, this is my first contribution, I make sure to make it simple.